rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -641,7 +635,7 @@ private static StringBuilder makeLink(StringBuilder b, String uriPrefix, String
         return b.append(link);
     }
 
-    private URI enlistCompensator(URI lraUri, long timelimit, String uriPrefix,
+    private URI enlistCompensator(URI lraUri, Long timelimit, String uriPrefix,
```

Comment:
An idea: what about rather change the `joinLRA` method to be using `long` instead of `Long`?
(as we are not in 1.0 we can dare to change the public api if agreed it's desirable)

Reply:
Well, I tend to prefer the `Long` as it allows the user to send `null` which is usually the _I don't care_ equivalent other than having to know beforehand that `0L` means wait indefinitely.

Reply:
I don't care could be interpreted to mean use a random timeout ;-)
BTW this is a private method so do we really care, the public API is already using the Long datatype:

> public URI joinLRA(URI lraId, Long timelimit, ...